src/data-expander.ts

```diff
@@ -220,8 +219,11 @@ export function globalVariables (gitlabData: any) {
 }
 
 export function flattenLists (gitlabData: any) {
-    traverse(gitlabData, ({parent, key, value}) => {
-        if (!Array.isArray(value) || parent == null || typeof key != "string") return;
-        parent[key] = value.flat(5);
-    });
```

Comment:
optional, i found out that disabling the
`cycleHandling` option should fixes the issue as well.

not too sure what you would prefer.